modules/openid_auth/spec/requests/validation_request_spec_v2.rb

```diff
@@ -287,17 +287,15 @@
   context 'with client credentials jwt and failed launch lookup' do
     before do
       allow(JWT).to receive(:decode).and_return(client_credentials_jwt)
-      allow(RestClient).to receive(:get).and_return(failed_launch_response)
+      allow(RestClient).to receive(:get).and_raise(failed_launch_response)
     end
 
     it 'v2 POST returns true if the user is a veteran' do
```

Comment:
Update this test description to reflect the expected behavior (returns a 401 when launch lookup fails).